version.py

```diff
@@ -33,9 +33,7 @@
 # Changes should be accompanied by documentation explaining what the
 # new level means in api_docs/changelog.md, as well as "**Changes**"
 # entries in the endpoint's documentation in `zulip.yaml`.
-
-API_FEATURE_LEVEL = 270  # Last bumped for direct_message_permission_group
-
+API_FEATURE_LEVEL = 271
```

Comment:
I realized I forgot to advertise the thing I was experimenting with here; see https://chat.zulip.org/#narrow/stream/412-api-documentation/topic/API_FEATURE_LEVEL.20merge.20conflicts/near/1883325.

Reply:
I updated that comment now.